### PR TITLE
** DO NOT MERGE** New Implementations

### LINKED ISSUE #74: Strategies to add for axelrod version > `3.11.0`
- `RichardHufford: K47R` : https://github.com/Axelrod-Python/Axelrod/pull/1162

### PR DESCRIPTION
**Pull request by meatballs**
Closes #74

### CODE REVIEW

**Comment by drvinceknight** [comment]
@meatballs I've just pushed e163730 let me know if you'd rather I left that to you to avoid confusion in the future :+1:

**Comment by meatballs** [comment]
All good! 

On 30 January 2018 07:52:46 GMT+00:00, Vince Knight <notifications@github.com> wrote:
>@meatballs I've just pushed e163730 let me know if you'd rather I left
>that to you to avoid confusion in the future :+1:

-- 
Owen Campbell 

This email has been digitally signed. 
For further information visit http://owencampbell.me.uk/pgp

**Comment by drvinceknight** [comment]
@meatballs I believe this is good to be released now but please check I haven't missed something :) (ZERO rush)

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def read(*names, **kwargs):
 
 setup(
     name='axelrod-fortran',
-    version='0.4.2',
+    version='0.4.3',
```
Comment (meatballs) [inline setup.py]: No need for this change - it's handled automatically by bumpversion

**Pull request merged**